netlify.toml

```diff
@@ -28,7 +28,7 @@
   [plugins.inputs]
     username = 'Stanford'
     password = 'OnPurpose'
-    context = ['branch-deploy', 'deploy-preview', 'production']
```

Comment:
I temporarily removed this as the 404 breaks when it is behind basic auth on Netlify. See former comment url.

Reply:
https://answers.netlify.com/t/401-unauthorized-trying-to-download-404-page/95492/11